pkg/leanix/integration_hub.go

```diff
@@ -123,13 +123,15 @@ func validateBindingKey(bindingKey BindingKey) error {
 	return nil
 }
 
-func validateConnectorConfiguration(configuration ConnectorConfiguration) error {
-	if configuration.ResolveStrategy == "" {
-		return fmt.Errorf("INVALID CONNECTOR CONFIGURATION: RESOLVE STRATEGY CANNOT BE EMPTY")
-	}
+func validateConnectorConfiguration(configuration ConnectorConfiguration, connectorId string) error {
+	if connectorId != "leanix-k8s-v3-connector" {
```

Comment:
For older connectors we are not doing the validation anymore?

Reply:
We are only doing it for old connectors. The new ones wont have this parameter anymore thats why we need this filter